test/unit/src/site/_utils/add-pagination/index.js

```diff
@@ -0,0 +1,35 @@
+const assert = require('assert');
+const addPagination = require('../../../../../../src/site/_utils/add-pagination');
+
+describe('add-pagination', function() {
+  describe('addPagination', function() {
```

Comment:
mind adding a test for the `additionalData` argument that `addPagination()` takes?